common/tests/fixtures/cardano/sign_tx.json

```diff
@@ -1502,15 +1502,7 @@
         "withdrawals": [
           {
             "path": "m/1852'/1815'/0'/2/0",
-            "amount": "1000"
```

Comment:
Hm, I see, so withdrawals actually can't come from multiple accounts now so this test would fail, right? That begs the question if the sorting is actually needed then, but I think let's keep it there in case the specs regarding staking paths change at some later point. But I think we can leave out the single withdrawal from the test case then.

Reply:
Yes, exactly.

> let's keep it there in case the specs regarding staking paths change at some later point

I agree.

> we can leave out the single withdrawal from the test case then

Ok, removed.